security/providers/oidc-common/src/main/java/io/helidon/security/providers/oidc/common/OidcConfig.java

```diff
@@ -641,38 +505,43 @@ public String paramName() {
     }
 
     /**
-     * Whether to use cooke to get the information from request.
+     * Tenant query parameter name.
      *
-     * @return if cookie should be used
-     * @see Builder#useCookie(Boolean)
+     * @return name of the tenant query parameter to use
+     * @see Builder#paramNameTenant(String)
      */
-    public boolean useCookie() {
-        return useCookie;
+    public String paramNameTenant() {
```

Comment:
Use approach similar to `cookie` - e.g. this method should be named `tenantParamName`

Reply:
renamed